tests/BUILD.bazel

```diff
@@ -1,17 +1,41 @@
 load("@bazel_skylib//rules:build_test.bzl", "build_test")
-load("//tools/bazel_integration_test:bazel_integration_test.bzl", "bazel_integration_test")
+load("@rules_bazel_integration_test//bazel_integration_test:defs.bzl", "default_test_runner")
+load(":integration_test.bzl", "rules_python_integration_test", "rules_python_integration_test_suite")
 
 package(default_visibility = ["//visibility:public"])
 
 licenses(["notice"])
 
-bazel_integration_test(
+_LEGACY_FLAGS = [
+    "--noenable_bzlmod",
+]
+
+default_test_runner(
+    name = "legacy_test_runner",
+    bazel_cmds = [
+        "info {}".format(" ".join(_LEGACY_FLAGS)),
+        "test {} //...".format(" ".join(_LEGACY_FLAGS)),
+    ],
+)
+
+default_test_runner(
+    name = "simple_test_runner",
```

Comment:
Just for posterity, out of scope for this PR:

Customizing the test runner to run a local pypi index is what I was thinking for how to run that local pypi server for testing the pypi integration. e.g. something like:

```
integration_test(
  runner = ":pypi_runner",
  pypi_packages = [":foo", ":bar"]
)

test_runner(
  name = "pypi_runner",
  src = "pypi_runner.sh",
  data = [":pypi_server"]
)

# pypi_runner.sh

$runfiles/pypi_server --index=$runfiles/pypi_packages
trap exit kill $?
$bazel $command $args
```

...something like that, anyways. Might be easier to implement it as Python code instead of bash because of windows. I think there's an example on the bazel_integration_test docs for how to write a runner in Python

Reply:
Yeah! That'll be awesome. I _think_ it's doable with the new rules.